test/Microsoft.AspNet.Mvc.Razor.Test/RazorPageCreateTagHelperTest.cs

```diff
@@ -57,6 +57,33 @@ public void CreateTagHelper_ProvidesTagHelperTypeWithViewContext()
         }
 
         [Fact]
+        public void CreateTagHelper_ProvidesTagHelperWithViewData()
+        {
+            // Arrange
+            var instance = CreateTestRazorPage();
+
+            // Act
+            var tagHelper = instance.CreateTagHelper<ViewDataTagHelper>();
+
+            // Assert
+            Assert.NotNull(tagHelper.ViewData);
+        }
+
+        [Fact]
+        public void CreateTagHelper_ProvidesTagHelperWithInternalProperty()
```

Comment:
though tiny nit: name should end with "ies" (not just one property involved here)